control/ctrlutil.py

```diff
@@ -45,6 +45,11 @@
 import numpy as np
 from numpy import pi
 
+# Hack for sphinx.ext.autodoc: if numpy is a mock import, then numpy.pi 
+# will be assigned to _Mock() and this generates a type error
+if not isinstance(pi, float):
+    pi = 3.14
```

Comment:
how about "from math import pi" here?  Or does that result in a mocked instance of some sort too?

Reply:
Much better fix since math is included in the base python distribution.